extensions/sql-database-projects/src/models/project.ts

```diff
@@ -242,23 +251,39 @@ export class Project {
 	}
 
 	private addDatabaseReferenceToProjFile(entry: DatabaseReferenceProjectEntry): void {
```

Comment:
i think this one should have some tests - lot of logic here

Reply:
yup I added tests for it in #10837